draft-ietf-rtcweb-jsep.xml

```diff
@@ -2315,7 +2315,42 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         bounds, or cannot be applied, processing MUST stop and an error MUST be
         returned.
           <list style="symbols">
-            <t>TODO</t>
+            <t>Basic sanity checking is applied for the following attributes in the media section:
+              <list style="symbols">
+                <t>ICE lite: MUST NOT be present.</t>
+                <t>ICE ufrag: MUST be present, and MUST meet the length criteria from [TODO: ref]</t>
+                <t>ICE pwd: MUST be present, and MUST meet the length criteria from [TODO: ref]</t>
+                <t>DTLS fingerprint: MUST be present, and, for existing media sections, MUST be unchanged.</t>
```

Comment:
If a new fingerprint is added, the browser would need a way of finding the cert and private key for that fingerprint so unless we want to get into all that, we might want to just say it has to be one of the fingerprints that was in the original.